src/pages-content/pricing-data.js

```diff
@@ -151,15 +151,6 @@ const plans = {
                 type: 'url',
                 value: 'https://posthog.com/docs/deployment',
             },
-            benefits: [
```

Comment:
With the way the pricing page is structured this breaks the page when you click on open source. It tries to look for benefits to display and fails.

Reply:
ha yes I pushed changed right when you commented this